bike_kollective/lib/bike_details_screen.dart

```diff
@@ -3,10 +3,11 @@ import 'package:flutter_riverpod/flutter_riverpod.dart';
 import 'package:bike_kollective/data/model/bike.dart';
 import 'bike_details_view.dart';
 import 'report_issue_dialog.dart';
+import 'package:bike_kollective/data/model/bike_with_distance.dart';
 
 // Define the BikeDetailsScreen widget
 class BikeDetailsScreen extends ConsumerWidget {
-  final BikeModel bike;
+  final BikeWithDistanceModel bike;
```

Comment:
Let's change this to "BikeWithDistance" that is simply a class with two members, the BikeModel and the distance.